decidim-core/app/mailers/concerns/decidim/multitenant_asset_host.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A module to be included in mailers that changes the default behaviour so
+  # the emails are rendered in the user's locale instead of the default one.
```

Comment:
I think this documentation does not correspond to this module :wink:

Reply:
Yep silly mistake :/